### PR TITLE
fixed dependency issue, scope issue, and stray comma

### PR DESCRIPTION
**Pull request by khenisey**

### CODE REVIEW

File: model/data.wl [hunk 1]
```diff
@@ -370,20 +370,28 @@ usStateDistancingPrecompute = Module[{
...
+    Prepend[
+      Map[
+        Min[1,(1+(#["retail_and_recreation_percent_change_from_baseline"]+#["transit_stations_percent_change_from_baseline"]+#["workplaces_percent_change_from_baseline"])/300. )]&,
+        Select[googleMobility,#["country_region_code"]=="US"&&#["sub_region_1"]==stateName&&#["sub_region_2"]==""&]],
+      state]
   ];
 
-  rawCsvTable=Prepend[stateDistancing[#]&/@Keys[fitStartingOverrides],dates];
-  
+  stateList={"AZ","CA","FL","PA","CO","TX","WA","CT","OH","NY","VA","VT","LA","MI","MS","MA","MD","GA","NJ","IL","IN","OK","WI","NV","OR","SC"};
```
Comment (khenisey) [inline model/data.wl:390]: Fix 1

File: model/data.wl [hunk 2]
```diff
@@ -465,7 +473,9 @@ usStateDistancingPrecompute = Module[{
       "distancingDays"->fullDays,
       "distancingLevel"->distancingLevel,
       "distancingData"->fullDistancing,
-      "distancingFunction"->distancingFunction|>
+      "distancingFunction"->distancingFunction,
+      "mostRecentDistancingDay"->DateString[DatePlus[DateObject[{2020,1,1}], Last[dataDays]]]
```
Comment (khenisey) [inline model/data.wl:477]: Fix 2 (part 1)

File: model/model.wl [hunk 3]
```diff
@@ -94,7 +94,8 @@ susceptibilityInitialPopulations=ConstantArray[1/susceptibilityBins,susceptibili
 
 
 
-mostRecentDistancingDay=DateString[DatePlus[DateObject[{2020,1,1}],Last[QuantityMagnitude[DateDifference[DateObject[{2020,1,1}],DateObject[#["date"]]]]&/@Select[googleMobility,#["country_region_code"]=="US"&&#["sub_region_1"]=="California"&&#["sub_region_2"]==""&]]-1]];
+(*mostRecentDistancingDay=DateString[DatePlus[DateObject[{2020,1,1}],Last[QuantityMagnitude[DateDifference[DateObject[{2020,1,1}],DateObject[#["date"]]]]&/@Select[googleMobility,#["country_region_code"]=="US"&&#["sub_region_1"]=="California"&&#["sub_region_2"]==""&]]-1]];*)
+mostRecentDistancingDay = usStateDistancingPrecompute["CA"]["scenario1"]["mostRecentDistancingDay"];
```
Comment (khenisey) [inline model/model.wl:98]: Fix 2 (part 2)

File: model/model.wl [hunk 4]
```diff
@@ -501,9 +502,9 @@ integrateModelSim[parameterizedSolution_, outputODE_, simulationParameters_]:=Mo
     outputSolutionRules,
     outputEvents,
     time,
-    soln,
+    soln
```
Comment (khenisey) [inline model/model.wl:505]: Fix 3

**Comment by wbunting** [comment]
👍

**Pull request merged**